packages/InteractiveProfilingTool-Core.package/IPTStats.class/instance/getWindowSpecWith..st

```diff
@@ -0,0 +1,13 @@
+toolbuilder
+getWindowSpecWith: aBuilder
+
+	| spec |
+
+	spec := self buildWindowWith: aBuilder specs: {
+		(0@0 corner: 1@1) -> [ self getTextSpecWith: aBuilder ]}.
+		
+	spec
+		label: 'Stats';
+		extent: 300@400.
```

Comment:
Squeak best-practice for dealing with magic numbers is return them from functions. (Same in line 7)

Reply:
Die magic numbers wurden jeweils in einer seperaten methode ausgelagert.